lib/ukrand/Config.uk

```diff
@@ -0,0 +1,56 @@
+menuconfig LIKUKRAND
+	bool "ukrand: Software random number generator"
+	default n
+
+if LIKUKRAND
+
+config LIBUKSWRAND
+	bool "Software-generated randomness"
+	default n
+
+if LIBUKSWRAND
+
+choice
+	prompt "Algorithm"
+	default LIBUKSWRAND_CHACHA
+
+config LIBUKSWRAND_MWC
+	bool "Multiply-with-carry"
+	help
+	  Use multiply-with-carry algorithm
+
+config LIBUKSWRAND_CHACHA
+	bool "ChaCha20"
+	help
+	  Use ChaCha20 algorithm
+endchoice
+
+choice
+	prompt "Initial seed"
+	default LIBUKSWRAND_INITIALSEED_TIME
+
+config LIBUKSWRAND_INITIALSEED_TIME
+	bool "Platform timestamp"
+
+config LIBUKSWRAND_INITIALSEED_RDRAND
```

Comment:
Should be `RDSEED`, not `RDRAND`. That is because the former provides higher brute-force resistance than the DRBG-generated `RDRAND`. Saying that, the `ukarch_random` API is architecture-independent, so you don't need to specify anything here. In conclusion what I would do is:
1. Remove all "Initial Seed" options
2. Make the entire library `depends on HAVE_RANDOM`.
3. In the implementation use the `ukarch_random_seed` to generate the seed if the system is configured to use a software PRNG.